addons/rook/template/base/install.sh

```diff
@@ -32,6 +32,15 @@ function rook_pre_init() {
             fi
         fi
     fi
+
+    # check Rook prerequisites
+    if rook_should_fail_install; then
+        bail "Rook ${ROOK_VERSION} will not be installed due to failed preflight checks."
+    fi
+
+    # Disable EKCO updates
```

Comment:
can we add more details to the comment to explain the reason for disabling as well as explain that it will be scaled back up when the ekco addon is applied?

Reply:
Extended comment to elaborate why EKCO is being disabled.